lib/templates.ts

```diff
@@ -563,6 +564,11 @@ export async function updateIsPublishedForTemplate(
           field: "isPublished",
         },
       ]);
+
+      if (templateWithAssociatedUsers.formRecord.isPublished)
```

Comment:
Nit: I don't think we drop the {} for if statements elsewhere but could be wrong
 
 ```javascript
 if (templateWithAssociatedUsers.formRecord.isPublished) {
        throw new TemplateAlreadyPublishedError();
 }
 ```

Reply:
It occurs on line 739 under the existing `removeDeliveryOption` function but happy to change it to always have {}. I don't think the linter catches it.